README.md

```diff
@@ -4,198 +4,25 @@
-</details>
-
-<details>
-<summary>Fully manual setup</summary>
-
-Completely manual setup is not recommended, only because of the need to install the npm packages in `setup.sh`, in the order they appear. I therefore recommend at least running that script. Manual setup using that script proceeds as follows:
-
-Clone the CadenceBot repository into a directory of your choosing.
-
-First, [install the latest `Node.js`](https://nodejs.org/en/download/) - You need at least version 12.0.0.
-
-Then, install the following:
-
- - `ffmpeg`
-
- - `gcc`
-
- - `g++`
-
-Now, run `setup.sh`, or open the file and install the npm packages present there in the order they appear.
-
-The next step is to setup your authentication file. [Get your token from Discord](https://github.com/reactiflux/discord-irc/wiki/Creating-a-discord-bot-&-getting-a-token), and place it in a file named `auth.json`, following the format in `auth-example.json`.
-
-Once this script finishes, [get your token from Discord](https://github.com/reactiflux/discord-irc/wiki/Creating-a-discord-bot-&-getting-a-token), and place it in a file called `auth.json` (see `auth-example.json`).
-
-At this point, you should be able to run CadenceBot, but in order to ensure that your `node_modules` stay up-to-date as new changes are made to CadenceBot, it is recommended that you install the `auto-setup.sh` script as a git hook.
-
-To do this, simply copy it into the `.git/hooks/` directory, as at least the `post-merge` hook. The `first-time-setup.sh` script copies my setup, which can is to have `auto-setup.sh` run on both post-merge and post-checkout. This can be duplicated with:
-
-`cp auto-setup.sh .git/hooks/post-checkout`
-
-`cp auto-setup.sh .git/hooks/post-merge`
-
-The `auto-setup.sh` script removes `node_modules` and re-runs `setup.sh`. If you wish, you can write your own script to duplicate this behavior, or simply skip it - This function is not necessarily required for CadenceBot to run, but it does avoid errors when new CadenceBot features rely on module updates, so if you don't keep your setup up-to-date automatically, be ready to have to update your modules as shown in `setup.sh` (keep in mind that this script does change on occasion).
-
-Now, CadenceBot should run properly, even if a change requires a new module or new module version. There is only one detail left - When CadenceBot is restarted by the `restart.sh` script, it keeps its log in the `CadenceBot.log` file, which is overwritten each time the script is run. If this is not desired, you can make a `maillog.sh` script, which shall be run before starting the bot each time `restart.sh` is run, with the intention of being used to archive the log before it is deleted (by email, usually). Add whichever commands you would like to this script to have your logs archived automatically.
-</details>
+## Implementation Details
```

Comment:
This feels like it could work at the end of the README, as a "You no longer care if you're just doing setup, you can read this part if you're looking for it instead of linking out"

To me, at least - I'm open to other opinions.